### ISSUE TITLE
TypeError on class_separation_plot of example notebook

### ISSUE THREAD

**Issue by itsoum** [statisticianinstilettos/recmetrics#14]
I attached the error below

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-30-05160122655c> in <module>
----> 1 recmetrics.class_separation_plot(pred_df, n_bins=45, class0_label="True class 0", class1_label="True class 1")

TypeError: class_separation_plot() got an unexpected keyword argument 'class0_label'
```

**Comment by otaviomguerra** [comment]
I'm experiencing the same error. It appears that this parameter doesn't exist anymore.

**Comment by gregwchase** [comment]
Fixed in #26
